packages/alfa-dom/test/node.spec.tsx

```diff
@@ -303,3 +305,239 @@ test(`#toJSON() serializes box of descendant inside content`, (t) => {
     Rectangle.of(16, 16, 50, 50).toJSON(),
   ]);
 });
+
+test("#toJSON() includes only serializationId when verbosity is minimal", (t) => {
+  const device = Device.standard();
+
+  const docId = crypto.randomUUID();
+  const elmId = crypto.randomUUID();
+  const attrId = crypto.randomUUID();
+
+  // We can't use JSX here because it doesn't support passing a serialization id when constructing an attribute
+  const doc = h.document(
+    [
+      h.element(
+        "div",
+        [h.attribute("id", "foo", undefined, attrId)],
+        undefined,
+        undefined,
+        undefined,
+        undefined,
+        undefined,
+        undefined,
+        elmId,
+      ),
+    ],
+    undefined,
+    undefined,
+    docId,
+  );
+
+  const options = {
+    device,
+    verbosity: json.Serializable.Verbosity.Minimal,
+  } as const;
```

Comment:
**Suggestion** (not blocking): It should be possible to group `Minimal` and `Low` with a `for (const verbosity of …)` loop in the test (same for nothing/`Medium`).